org.frescobaldi.Frescobaldi.yaml

```diff
@@ -0,0 +1,33 @@
+---
+app-id: org.frescobaldi.Frescobaldi
+
+# TODO:
+# - QtWebEngine is not included in org.kde.Sdk, so Documentation
+# Browser is not working. Should I really use this?
+# https://github.com/flathub/io.qt.qtwebengine.BaseApp
+# - Cannot hear MIDI playback (handled by a user systemd service in the host).
+# - Check if MIDI input works.
+
+runtime: org.kde.Platform
+runtime-version: '5.14'
```

Comment:
5.15 is now released.

Reply:
I cannot use 5.15 because it requires org.freedekstop.Platform=20.08 ,but I'm using the org.freedesktop.Sdk.Extension.texlive, which doesn't have 20.08 yet.

Reply:
ah right.

Reply:
It does have a 20.08 now.

Reply:
In [this comment](https://github.com/flathub/flathub/pull/1277#issuecomment-694156382) I suggested waiting for next lilypond stable release (scheduled in 1-2 months).

Reason: guile1.8 is legacy software and need patches to be built using 20.08. I might ask for help on lilypond-devel list. If I manage to build guile1.8, fine. Otherwise I'll bit the bullet and build guile2 (significantly slower), which is supported only in next lilypond stable.

Next Monday I'll make a try.